extension/chrome/src/popup.html

```diff
@@ -64,6 +64,10 @@
 					<label for="template_text">Template:</label>
 					<textarea class="form-control" rows="10" id="template_text"></textarea>
 				</div>
+				<div class="form-group">
+					<label for="template_url">Template URL:</label>
+					<input class="form-control" id="template_url" type="text"></input>
```

Comment:
[`<input>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) does NOT have a closing tag in HTML

Reply:
so this should be changed in all places, I just copy-pasted code above :))